tests/sip-sdp/test.yaml

```diff
@@ -20,11 +20,73 @@ checks:
         sip.sdp.phone_number: +1 617 555-6011 (Jane Doe)
         sip.sdp.connection_data: IN IP4 192.168.1.2
         sip.sdp.bandwidths[0]: AS:64
-        sip.sdp.time: 3034423619 3042462419
-        sip.sdp.repeat_time: 604800 3600 0 90000
+        sip.sdp.time_descriptions[0].time: 3034423619 3042462419
+        sip.sdp.time_descriptions[0].repeat_time: 604800 3600 0 90000
         sip.sdp.timezone: 2882844526 -1h 2898848070 0
         sip.sdp.encryption_key: prompt
         sip.sdp.attributes[0]: sendrecv
         sip.sdp.media_descriptions[0].media: audio 30000 RTP/AVP 0 8 97 2 3
+        sip.sdp.media_descriptions[0].media_info: media info
+        sip.sdp.media_descriptions[0].connection_data: IN IP4 192.168.1.2
+        sip.sdp.media_descriptions[0].encryption_key: prompt
         sip.sdp.media_descriptions[0].attributes[0]: rtpmap:0 pcmu/8000
-
+  - filter:
+      count: 1
+      match:
+        event_type: alert
+        alert.signature_id: 1
```

Comment:
nit you can test that sdp metadata is logged with the alert and matches

and other nit : you can test that some other signatures do not match §so that we do not have FPs)